meteor-app/imports/api/assessments/parts.js

```diff
@@ -2,7 +2,7 @@ import { Mongo } from 'meteor/mongo'
 import SimpleSchema from  'simpl-schema'
 import { RegExId, createdAt, updatedAt } from '/imports/api/schema'
 
-const Parts = new Mongo.Collection('partsData')
+const Parts = new Mongo.Collection('ServicesItem')
```

Comment:
Chris, thanks for making the change, but it's not quite right.
collection names must be plural and camelCase, eg `ServicesItem` => `serviceItems`
The variable you use to refer to the collection must be PascalCase, eg `Parts` => 'ServiceItems`.
You will now need to update the code that refers to `Parts` to match this.
Cheers
Mike